src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java

```diff
@@ -436,7 +436,7 @@ private void createLine() throws LineUnavailableException, StreamPlayerException
 			// Calculate the Sample Size in bits
 			int nSampleSizeInBits = sourceFormat.getSampleSizeInBits();
 			if (sourceFormat.getEncoding() == AudioFormat.Encoding.ULAW || sourceFormat.getEncoding() == AudioFormat.Encoding.ALAW
-				|| nSampleSizeInBits <= 0 || nSampleSizeInBits != 8)
```

Comment:
I think this is incorrect here it removed nSampleSizeInBits <= 0 which is needed .

Reply:
If (nSampleSizeInBits <= 0) is true, then also (nSampleSizeInBits != 8) is true.

Therefore (nSampleSizeInBits <= 0) is redundant.

Reply:
Hm , i didn't wrote that logic so i am not completely sure why it was there.